hat/assets/js/apps/Iaso/domains/registry/components/Instances.tsx

```diff
@@ -91,6 +94,29 @@ export const Instances: FunctionComponent<Props> = ({
     const currentForm: Form | undefined = useMemo(() => {
         return formsList?.find(f => `${f.value}` === formIds)?.original;
     }, [formIds, formsList]);
+
+    useEffect(() => {
+        if (
+            formsList?.length &&
+            formsList?.length >= 0 &&
+            !isFetchingForms &&
+            !formIds &&
+            orgunitTypeDetail
+        ) {
+            let selectedForm = formsList[0].value;
+            if (orgunitTypeDetail.reference_forms.length > 0) {
+                selectedForm = `${orgunitTypeDetail.reference_forms[0].id}`;
+            }
```

Comment:
```suggestion
         const selectedForm = orgunitTypeDetail.reference_forms.length > 0
             ? `${orgunitTypeDetail.reference_forms[0].id}`
             : formsList[0].value;
```

(You might want to pass that through a proper linter)